File: app/src/main/java/com/example/coffea/MainContributingActivity.java
package com.example.coffea;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class MainContributingActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main_contributing);
    }



    public void toContributionChecklist (View view)
    {
        Intent goToContributionChecklist = new Intent (MainContributingActivity.this, ContributionCheckList.class);
        startActivity(goToContributionChecklist);
    }
}

File: app/src/main/java/com/example/coffea/MainContributingActivity.kt
package com.example.coffea;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class MainContributingActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main_contributing);

        //Hide the navigation bar
        hideNavigationBar();
    }

    //Hide navigation bar even after going out and coming back to the app
    @Override
    protected void onResume()
    {
        super.onResume();
        hideNavigationBar();
    }


    //Hide the navigation bar
    private void hideNavigationBar ()
    {
        this.getWindow().getDecorView().setSystemUiVisibility(
                View.SYSTEM_UI_FLAG_HIDE_NAVIGATION |
                        View.SYSTEM_UI_FLAG_IMMERSIVE_STICKY

        );




    }



    public void toContributionChecklist (View view)
    {
        Intent goToContributionChecklist = new Intent (MainContributingActivity.this, ContributionCheckList.class);
        startActivity(goToContributionChecklist);
    }
}